website/content/docs/platform/k8s/csi/configurations.mdx

```diff
@@ -117,6 +125,8 @@ structure is illustrated in the [examples](/docs/platform/k8s/csi/examples).
 
   - `secretKey` `(string: "")` - The key in the Vault secret to extract. If omitted, the whole response from Vault will be written as JSON.
 
+  - `filePermission` `(integer: 0o644)` - The file permissions to set for this secret's file.
```

Comment:
Do we support setting this in octal syntax? (If not, this could be confusing.)

Reply:
Good question, just tested this with the e2e tests locally and it works fine.